File: windows/db_explorer/modules/drivers/models/DatabaseQueryResult.cs
#pragma warning disable IDE1006 // JS Naming Conventions

namespace db_explorer.modules.drivers.models
{
    public struct DatabaseQueryResult
    {
        public DatabaseColumnInfo[] columns { get;  set; }
        public string[][] rows { get; set; }
        public long affectedRows { get; set; }
    }
}

File: android/app/src/main/kotlin/com/db_explorer/modules/drivers/models/DatabaseQueryResult.kt
package com.db_explorer.modules.drivers.models

class DatabaseQueryResult(
    _rowsAffected: Long,
    _columns: Array<DatabaseColumnInfo>,
    _rows: Array<Array<String>>,
) {
    var rowsAffected: Long = _rowsAffected
    var columns: Array<DatabaseColumnInfo> = _columns
    var rows: Array<Array<String>> = _rows
}
